core/src/main/res/values-tr/strings.xml

```diff
@@ -276,4 +276,6 @@
   <string name="crash_checkbox_file_system">Dosya Sistemi Ayrıntıları</string>
   <string name="diagnostic_report">Teşhis Raporu</string>
   <string name="diagnostic_report_message">Sorunu teşhis edebilmemiz için lütfen aşağıdaki tüm ayrıntıları gönderin</string>
+  <string name="percentage">%%d%</string>
```

Comment:
@Abijeet this will cause a crash because it is an invalid format string, I don't know the tr convention for showing percentages but it would want to be `%%%d` to show "%5" or `%d%%` to show "5%"